### PR TITLE
Algorithm to create an AudioSession

### PR DESCRIPTION
**Pull request by youennf**
This PR introduces how an AudioSession object is created and how its attributes are working.

***
<a href="https://pr-preview.s3.amazonaws.com/youennf/audio-session/pull/30.html" title="Last updated on Oct 22, 2024, 1:36 PM UTC (cf1ccea)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/audio-session/30/66d2ba3...youennf:cf1ccea.html" title="Last updated on Oct 22, 2024, 1:36 PM UTC (cf1ccea)">Diff</a>

### CODE REVIEW

**Comment by youennf** [comment]
This PR refers to algorithms defined in https://github.com/w3c/audio-session/pull/23.
The links will only fully work when both PRs are merged.

**Comment by youennf** [comment]
Drive-by fix of https://github.com/w3c/audio-session/issues/21

File: index.bs [hunk 1]
```diff
@@ -76,7 +76,33 @@ It is accessed through the {{Navigator}} interface (see [[#extensions-to-navigat
   };
 </pre>
 
-## Audio session states ## {#audio-session-types}
+To create an {{AudioSession}} object in |realm|, run the following steps:
+
+1. Let |audioSession| be a new {{AudioSession}} object in |realm|, initalized with the following internal slots:
```
Comment (alastor0325) [inline index.bs:81]: s/initalized/initialized

**Review by alastor0325 — APPROVED** [review]
LGTM, thanks!

File: index.bs (hunk 1 above)
Comment (youennf) [reply index.bs]: Done

**Pull request merged**